packages/augur-ui/src/modules/create-market/components/form.tsx

```diff
@@ -408,7 +408,7 @@ export default class Form extends React.Component<FormProps, FormState> {
         ? isLessThan(value, readableName, newMarket.maxPrice, lessThanMessage)
         : '',
       checkDividedBy ? dividedBy(value, readableName, newMarket.minPrice, newMarket.maxPrice) : '',
-      checkGreaterDate ? dateGreater(value, currentTimestamp) : '',
+      checkGreaterDate && newMarket.endTimeFormatted ? dateGreater(newMarket.endTimeFormatted.timestamp, currentTimestamp) : '',
```

Comment:
this should be generic

Reply:
since the property is aggregated from all date components that wouldn't be possible. will add conditional as you suggested. if (label === "setEndTime") ...